src/tools/gxpkg.ss

```diff
@@ -340,7 +339,13 @@
       (force once)
       (force +pkg-root-dir+))))
 
-(def (pkg-new prefix name maybe-link)
+(def (pkg-new package-prefix package-name maybe-link)
+  (def prefix (or package-prefix
+                  (getenv "USER" #f)
+                  (begin (eprintf "Package prefix not specified with -p or --package, and USER not defined\n")
+                         (exit 2))))
```

Comment:
that's a little rude.

I think we should just raise an error with the informative message, instead of print and exit.

Reply:
I think there should be an `(abort ...)` primitive that throws an error that is specially caught by the runtime to print a message and exit without a stack trace (unless stack trace is specifically requested by a parameter typically set from a command-line option).

See `:clan/exit` for some of the beginnings of that.

Until then, what should we do?

Reply:
Just error it and we can add abort to std/error

Reply:
Where in the runtime is the right place to add the stuff in clan/exit ?

Reply:
It doesn't need to be in runtime, we jist need an Exception object that defines display-exception, it can live in std/error.